errlist/errlist.go

```diff
@@ -2,19 +2,20 @@ package errlist
 
 import (
 	"bytes"
-	"fmt"
+
+	"github.com/pkg/errors"
 )
 
-type ErrList []error
+type ErrList []FieldErr
```

Comment:
Really I don\`t like name `errlist.ErrList`. Why not just `errlist.List`. Also why `AddFieldErr` not just `Add`, `Push` or `Append`. I think it\`s better to use some term more related to list.